roles/contiv_network/templates/netmaster.j2

```diff
@@ -1 +1 @@
-NETMASTER_ARGS="--cluster-mode {{netplugin_mode}} -cluster-store {{cluster_store}} "
+NETMASTER_ARGS="--cluster-mode {{netplugin_mode}} -cluster-store {{cluster_store}} -listen-url :{{ netmaster_port }}"
```

Comment:
per @DivyaVavili we need some non-trivial changes before we can make the port configurable. We have https://github.com/contiv/netplugin/issues/705 to track this, but won't be done by beta time frame.